handler.js

```diff
@@ -277,13 +277,57 @@ export function ingest({ Records: records }: Object, context: Object) {
           return parse(raw)
             .then(mapToAirings)
             .then(function(airings) {
-
               console.log('Parsed airings list', airings.length);
 
+              let airingData = {};
```

Comment:
`airingData` here can be assumed to only need to represent a single channel. In any given invocation of the ingest function, all airings will have the same channel value.

It can likely be reduced to:
```
let channel = airings[0].channel
```

It would need a single conditional though checking that there is at least one airing.